app.js

```diff
@@ -189,6 +189,11 @@ app.post('/api/classify', app.upload.single('images_file'), function(req, res) {
       if (req.body.classifier_id) {
         combine.value[0].classifier_ids = req.body.classifier_id;
       }
+
+      combine.value[0].raw = {};
+      methods.map(function(methodName, idx) {
+        combine.value[0].raw[methodName] = encodeURIComponent(JSON.stringify(results[idx].value[0]));
+      });
       res.json(combine.value[0]);
```

Comment:
pretty-printing the json broke the underscore templates in a way I didn't understand.  That's also the thing that complicates a refactoring of this method,  it's pretty tightly coupled to the underscore templates.  I'd rather refactor it all (templates and endpoint), but that's not something that can reasonably be done in this sprint.